core/authority_discovery/publisher/address_publisher.cpp

```diff
@@ -160,21 +162,23 @@ namespace kagome::authority_discovery {
       addresses.emplace(std::move(address2));
     }
     metric_amount_addresses_last_published->set(addresses.size());
-    ::authority_discovery_v3::AuthorityRecord record;
+    ::authority_discovery_v2::AuthorityRecord record;
```

Comment:
No need to change protobuf file, disabling two `if`s is sufficient
```suggestion
    ::authority_discovery_v3::AuthorityRecord record;
```